go-bindata-assetfs/main.go

```diff
@@ -47,21 +45,30 @@ func main() {
 			done = true
 		}
 	}
+	/* Note by khaos: Prefix omitted due to strange behaviour. */
 	fmt.Fprintln(out, `
 func assetFS() *assetfs.AssetFS {
-	for k := range _bintree.Children {
-		return &assetfs.AssetFS{Asset: Asset, AssetDir: AssetDir, Prefix: k}
+	for _ = range _bintree.Children {
+		return &assetfs.AssetFS{Asset: Asset, AssetDir: AssetDir, Prefix: ""}
 	}
 	panic("unreachable")
 }`)
+	// Close files BEFORE remove calls (don't use defer).
+	in.Close()
+	out.Close()
 	if err := os.Remove("bindata.go"); err != nil {
 		fmt.Fprintln(os.Stderr, "Cannot remove bindata_assetfs.go", err)
 	}
 }
 
 func isInPath(filename string) bool {
```

Comment:
This entire `isInPath` func looks suspiciously close to [`exec.LookPath`](http://godoc.org/os/exec#LookPath), any reason not to use that?